README.md

```diff
@@ -13,16 +13,22 @@ The Search Hacker News React + Redux app example in this repo is based on [Robin
 You can try out the Search Hacker News app [here](http://fs-redux-sentry.s3-website-us-east-1.amazonaws.com/) or you can clone this repo and `npm install` then `npm run start`. The code is built with [Create React App](https://github.com/facebook/create-react-app).
 
 ### Setup
+First, copy the file `.env_sample` to `.env`. You will need to fill in those values to set up Sentry and FullStory correctly.
+
 You’ll need a [FullStory account](https://www.fullstory.com/pricing/) and a [Sentry account](https://sentry.io/signup/). Sentry and FullStory should be initialized as soon as possible during your application load up. In Search Hacker News, `initSentry` and `FullStory.init` are called before the [`App`](https://github.com/patrick-fs/fs-react-redux-sentry/blob/master/src/components/App.js) component is loaded in [src/index.js](https://github.com/patrick-fs/fs-react-redux-sentry/blob/master/src/index.js).
 
```

Comment:
Would it be a good idea to mention the required settings changes in Sentry? https://github.com/getsentry/sentry-fullstory#sentry-settings-change

Reply:
@patrick-fs Good catch, yes